pkgs/games/hex-a-hop/default.nix

```diff
@@ -0,0 +1,41 @@
+{lib, stdenv, makeDesktopItem, copyDesktopItems, fetchurl, SDL, SDL_mixer, SDL_ttf}:
+
+stdenv.mkDerivation rec {
+   pname = "hex-a-hop";
+   version = "1.1.0";
+
+   src = fetchurl {
+     url = "https://downloads.sourceforge.net/project/hexahop/1.1.0/hex-a-hop-1.1.0.tar.gz";
+     sha256 = "sha256-gL8Z2TaoQwyrW8Ro7mgnw4BS4CbGCet+zhYxfnkyoNY=";
+   };
+
+   buildInputs = [ SDL SDL_mixer SDL_ttf copyDesktopItems];
+
+   makeFlags = [ "PREFIX=$(out)" ];
+
+   desktopItems = makeDesktopItem rec {
+     name = pname;
+     exec = name;
+     icon = name;
+     desktopName = name;
+     comment = meta.description;
+     type = "Application";
+     categories = [ "Game" "LogicGame" ];
+     startupNotify = false;
+   };
+
+   postFixup = ''
+     install -Dm644 ${./hex-a-hop.png} $out/share/icons/${pname}.png
```

Comment:
Could we fetch this instead or, even better, take it out of src?

Reply:
I took it from the Arch Linux package so I guess we can take it from there. The source doesn't contain the png, not sure where it comes from.

Reply:
Nevermind, it is there, it's a bmp

Reply:
Please fetch it from there

Reply:
is [this](e0d3d5e04f2eee86ae6bf4501247f566f7c8126d) alright?

Reply:
is e0d3d5e04f2eee86ae6bf4501247f566f7c8126d alright?

Reply:
I also noticed that debian packages the icon in every size, what's the recommended way to do this in Nix?

Reply:
I'm not aware of any policy here. I'd just take the largest.